### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JulianFun123**
Made everything just cleaner :)

### CODE REVIEW

File: app/src/main/java/de/datlag/hotdrop/MainActivity.java [hunk 1]
```diff
@@ -235,60 +236,43 @@ private void initializeLogic() {
         settingsManager = new SettingsManager(activity, firebaseManager);
         storageManager = new StorageManager(activity, firebaseManager);
 
+        storageManager.downloadFile("V9g8NkdN9DQzd7rH3SMv");
```
Comment (DatL4g) [inline app/src/main/java/de/datlag/hotdrop/MainActivity.java]: remove this

File: app/src/main/java/de/datlag/hotdrop/MainActivity.java [hunk 2]
```diff
@@ -404,67 +374,57 @@ public CoordinatorLayout getCoordinatorLayout() {
...
         super.onActivityResult(requestCode, resultCode, data);
         googleAuth.onActivityResult(requestCode, resultCode, data);
     }
 
-    @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
     }
+
+
```
Comment (DatL4g) [inline app/src/main/java/de/datlag/hotdrop/MainActivity.java]: unnecessary spaces

File: app/src/main/java/de/datlag/hotdrop/firebase/StorageManager.java [hunk 3]
```diff
@@ -203,7 +195,43 @@ private String randomString(int sizeOfRandomString)
 
     public interface FileUploadCallback{
         void onSuccess(String url);
-
         void onFailure(Exception exception);
     }
+
+    public void downloadFile(String docId){
+
+        Log.d("ROFL", "uploadData/normal/" + getUser().getUid());
```
Comment (DatL4g) [inline app/src/main/java/de/datlag/hotdrop/firebase/StorageManager.java]: remove this

**Comment by DatL4g** [comment]
add removed @Override Annotations

**Comment by JulianFun123** [comment]
nope

**Pull request merged**